vmware_aria_operations_integration_sdk/mp_build.py

```diff
@@ -184,23 +183,14 @@ def registry_prompt(default: str) -> str:
 def get_container_registry(
     adapter_kind_key: str,
     config_file: str,
+    container_registry: Optional[str],
     container_registry_arg: Optional[str],
```

Comment:
Does it make sense to have both of these arguments? If so, we should rename so it's clear how they differ.

Does the function name still make sense? It seems we aren't getting the container registry, except in the case where it doesn't exist yet.

Reply:
ok, the renamed variables definitely make more sense, but I'm still not convinced we need both. It looks like there's a lot of duplicate code that could be handled by just sending the correct registry value to the function in the first place. There's already fallback logic for the two config keys, just add a new highest-priority case that gets the registry from the command line arguments.

I think if you're going to change the name of the function, it should be in this MR, as this is where the changes that require the new name is happening.